### PR TITLE
Enables non sudo users and allow setting default shell

### PR DESCRIPTION
**Pull request by bonzofenix**
This PR maintains current behavior while adding 2 optional fields for the user_add job.

### CODE REVIEW

File: jobs/user_add/templates/pre-start.sh.erb [hunk 1]
```diff
@@ -4,6 +4,8 @@ set -ex
 
 <% p('users').each do |user_hash| %>
   <%  user = user_hash['name']
+      sudo = user_hash['sudo'].nil? ? true : !!user_hash['sudo']
+      shell = user_hash['shell'].nil? ? 'bash' : user_hash['shell']
```
Comment (dpb587-pivotal) [inline jobs/user_add/templates/pre-start.sh.erb]: can you make this an absolute path so it's more explicit and closer to what `usermod` traditionally needs. also allows for a bit more flexibility.

Comment (bonzofenix) [inline jobs/user_add/templates/pre-start.sh.erb]: Do you mean the shell path `/bin/bash`? That sounds like a good idea actually.

Comment (dpb587-pivotal) [inline jobs/user_add/templates/pre-start.sh.erb]: yep

**Comment by dpb587-pivotal** [comment]
Merged - thanks!

**Pull request merged**